examples/add_image_feature_pipeline.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# Copyright 2022 Zegami Ltd
+from zegami_sdk.client import ZegamiClient
+
+WORKSPACE_ID = ''
+COLLECTION_ID = ''
+NAME = 'Image Similarity 2'
+
+zc = ZegamiClient()
+
+workspace = zc.get_workspace_by_id(WORKSPACE_ID)
+collection = workspace.get_collection_by_id(COLLECTION_ID)
+
+collection.add_feature_pipeline(
+    name='mRMR20',  # name would be used to derive column ids/titles
+    source=0,   # index of source to use
+    steps=[     # list of nodes which would feed one into the other in sequence
+        {
+            'action': 'mRMR',
+            'params': {
+                'target_column': 'lipid_volume',
+                'K': 20,
+            },
+        },
+        {
+            'action': 'cluster',
+            'params': {
+                "out_column_start_order": 1010,
```

Comment:
Agree it's not ideal for the user to have to supply this. We might be able to just find the column orders for existing nodes, and add a large number to it?
This is another problem that could potentially be solved/simplified by having the client load multiple data files in parallel, rather than having the merge node.

Reply:
That wouldn't work if the merge node isn't finished yet, and this is likely to get run immediately after creation sometimes.
Some other options I can think of:
* Keep track in `userdata` somehow, although I don't really like the thought of doing that.
* Do something like count the number of nodes currently in the image_clustering category and start at that x1000?
* Just start at 1 and have the merge deal with this by setting column order as 1000 * parent index